platform/xen/rumpxendev/xendev_component.c

```diff
@@ -57,6 +57,7 @@ static const struct xen_dev_info {
 	[cmin] = { path, init_fn, component##_dev_open,	\
 		   &component##_dev_fileops }
 	XDEV(0, DEV_XEN "/xenbus", NULL, xenbus),
+	XDEV(1, "/dev/xenevt", xenevt_dev_init, xenevt),
```

Comment:
On 22/05/15 08:59, Wei Liu wrote:
>> @@ -57,6 +57,7 @@ static const struct xen_dev_info {
>>   	[cmin] = { path, init_fn, component##_dev_open,	\
>>   		   &component##_dev_fileops }
>>   	XDEV(0, DEV_XEN "/xenbus", NULL, xenbus),
>> +	XDEV(1, "/dev/xenevt", xenevt_dev_init, xenevt),
>
> That's not doable as those devices are under different directories.

Ah, DEV_XEN is not "/dev"?  Interesting ;)